ArduCopter/AP_Arming.cpp

```diff
@@ -658,6 +658,11 @@ bool AP_Arming_Copter::mandatory_checks(bool display_failure)
         result = false;
     }
 
+#if AP_OPENDRONEID_ENABLED
+    // check OpenDroneID
+    result = opendroneid_checks(display_failure);
```

Comment:
This overwrites the false return value from either of the previous checks - so if the gps checks fail but the opendroneid checks pass then we'll allow flight....

Reply:
fixed